config/ng-config.json

```diff
@@ -82,7 +82,8 @@
       "customLocaleURL": "/bahmni_config/openmrs/i18n/",
       "loginPageUrl": "../../bahmni/home/index.html#/login",
       "patientsURL": "/bahmni/clinical/index.html#/default/patient/",
-      "patientsURLGeneralInformationTab": "/dashboard?currentTab=DASHBOARD_TAB_GENERAL_KEY"
+      "patientsURLGeneralInformationTab": "/dashboard?currentTab=DASHBOARD_TAB_GENERAL_KEY",
+      "patientDashboardURL": "/openmrs/coreapps/clinicianfacing/patient.page"
```

Comment:
@icrc-psousa I think this can be left as being more generic, it doesn't have to be a link to a patient dashboard, it just has to be a link driven by a patient external reference.

So let's rename it to something simply like `"appointmentsListPatientLink"`.

Then, we will need an aliasing mechanism so that in the end this would be configured as such:
```json
"appointmentsListPatientLink": "/openmrs/coreapps/clinicianfacing/patient.page?patientId ={{uuid}}"
```

@binduak, @buvaneswari-arun, @angshu, thoughts? How would you go about the aliasing syntax here?

Reply:
Yes. thats the syntax {{}} that we follow. all links are to be relative.